meinberlin/apps/budgeting/views.py

```diff
@@ -122,10 +123,8 @@ def post(self, request, *args, **kwargs):
             request.session["token_expire_date"] = (
                 datetime.datetime.now() + TOKEN_SESSION_EXPIRE
             ).timestamp()
-            kwargs["valid_token_present"] = True
-        kwargs["token_form"] = token_form
```

Comment:
I think we still need to pass the token form here in case it has errors, currently they dont show anymore... not sure about the valid_token_present, we check again what that this is used for